std_msgs/Header header
geometry_msgs/Vector3 wind_speed
